pkg/bootstrap/docker/openshift/ansible.go

```diff
@@ -0,0 +1,310 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path"
+	"text/template"
+
+	"github.com/golang/glog"
+
+	kapierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kbatch "k8s.io/kubernetes/pkg/apis/batch"
+	kclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/bootstrap/docker/host"
+)
+
+const (
+	defaultAnsibleImageUser        = "root"
+	defaultOpenshiftAnsibleImage   = "ansible"
+	deploymentTypeOrigin           = "origin"
+	deploymentTypeOCP              = "openshift-enterprise"
+	imageStreamCentos              = "centos7"
+	openshiftAnsibleServiceAccount = "openshift-ansible"
+)
+
+const defaultMetricsInventory = `
```

Comment:
This is officially making these all part of an API, which we kind of say anyway, but I want @sdodson to sign off on the implications that we are now creating a new public API that CANNOT BREAK EVER (or at least over long time).